.changeset/angry-roses-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/fe-fpm-writer': patch
+---
+
+Added `fragmentName` property for custom elements type to use it as key if defined in manifest templates
```

Comment:
changelog mentions property `fragmentName`, but in source code it is `fragmentFile`